Tools/clinic/clinic.py

```diff
@@ -1214,6 +1257,9 @@ def parser_body(
                     add_label = None
                 if not p.is_optional():
                     parser_code.append(normalize_snippet(parsearg, indent=4))
+                    if p.deprecated_positional:
+                        code = deprecate_positional_use(p)
+                        parser_code.append(code)
```

Comment:
Should not it be added before the parsing code?

Reply:
Yes, thanks for catching that.